yarn.lock

```diff
@@ -2824,10 +2824,9 @@ import-fresh@^3.0.0, import-fresh@^3.2.1:
     parent-module "^1.0.0"
     resolve-from "^4.0.0"
 
-import-in-the-middle@^1.8.1:
+"import-in-the-middle@https://github.com/DataDog/import-in-the-middle#dbaca128f6eb517d71bcf7e6b6b45f21a07128a4":
```

Comment:
hopefully we either remove this change or we contribute the changes to a released version of iitm